src/test/bridges/example/ExampleUnit.t.sol

```diff
@@ -2,49 +2,51 @@
 // Copyright 2022 Aztec.
 pragma solidity >=0.8.4;
 
+import {Test} from "forge-std/Test.sol";
 import {AztecTypes} from "../../../aztec/libraries/AztecTypes.sol";
-import {BridgeTestBase} from "./../../aztec/base/BridgeTestBase.sol";
 
 // Example-specific imports
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {ExampleBridgeContract} from "../../../bridges/example/ExampleBridge.sol";
 import {ErrorLib} from "../../../bridges/base/ErrorLib.sol";
 
-contract ExampleTest is BridgeTestBase {
+// @notice The purpose of this test is to directly test convert functionality of the bridge.
+contract ExampleUnitTest is Test {
     IERC20 public constant DAI = IERC20(0x6B175474E89094C44Da98b954EedeAC495271d0F);
 
+    AztecTypes.AztecAsset internal emptyAsset;
+
+    address private rollupProcessor;
     // The reference to the example bridge
-    ExampleBridgeContract internal bridge;
+    ExampleBridgeContract private bridge;
 
-    // To store the id of the example bridge after being added
-    uint256 private id;
+    // @dev This method exists on RollupProcessor.sol. It's defined here in order to be able to receive ETH like a real
```

Comment:
Would probably just say `RollupProcessor` without the `.sol` but not a big deal.